com.korbsstudio.penpot-desktop.yml

```
@@ -0,0 +1,36 @@
+app-id: com.korbsstudio.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://updates.korbsstudio.com/penpot-desktop/Penpot%20Desktop-0.1.1.AppImage
+        sha256: 48a1724a36f1e7d44d4c541f8470cceb67aa2a81fcd7280d1baaf2a9240eba50
+      - type: file
+        path: ./run.sh
+      - type: file
+        path: ./penpotdesktop.desktop
+      - type: file
+        path: com.korbsstudio.penpot-deskop.appdata.xml
+    build-commands:
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/icons/hicolor/128x128/apps/
+      - install -Dm755 ./run.sh /app/bin/run.sh
+      - chmod +x Penpot*.AppImage
+      - ./Penpot*.AppImage --appimage-extract
+      - mv squashfs-root /app/bin/penpot-desktop
+      - install -D penpotdesktop.desktop /app/share/applications/penpotdesktop.desktop
```

this is not the right name for the .desktop

```suggestion
      - install -D penpotdesktop.desktop /app/share/applications/${FLATPAK_ID}.desktop
```